brainstormforce/ultimate-addons-for-gutenberg
# Fix: Translation in ultimate addon for gutenberg.

### Description

- Trello - https://trello.com/c/IHfNFtin/56-translations-in-uag

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Install loco translate plugin.
- Change WP language from Settings > General and Users > Profile
- Go to Loco translate > Plugins > UAG > click on uag > click on add language btn > choose language > Save changes > Sync and translate string > Save.
- Check by drag and drop any block such as Review block.

### Checklist:
